tutorials/workflow/csharp/combined-patterns/WorkflowApp/OrderWorkflow.cs

```diff
@@ -0,0 +1,65 @@
+using Dapr.Workflow;
+using WorkflowApp.Activities;
+
+namespace WorkflowApp;
+
+internal sealed class OrderWorkflow : Workflow<Order, OrderStatus>
+{
+   public override async Task<OrderStatus> RunAsync(WorkflowContext context, Order order)
+   {
+       var inventoryTask = context.CallActivityAsync<ActivityResult>(
```

Comment:
Can you add some comments in here to break up where each pattern starts/stops? Ie. //Fan-in/fan-out